index.html

```diff
@@ -251,11 +265,12 @@ <h4 class="heading-langS">Skills</h4>
                                                      <li><a href="https://github.com/"><img src="resources\github (1).svg" alt="GitHub icon" /></a></li>
                                                      <li><a href="https://www.angellist.com/"><img src="resources\angellist.svg" alt="Angelist icon" /></a></li>
                                   </ul>
+                                                </div>
                                 </section>
                                               </div>
```

Comment:
[OPTIONAL]
- Kindly note that this closing `div` tag has no opening tag, kindly fix it. If its opening tag is at line `259`, you need to lift the closing tag before the closing `section` tag.